Samples/StyleTransfer/VideoEffect/StyleTransferEffectCpp/StyleTransferEffect.cpp

```diff
@@ -0,0 +1,94 @@
+﻿#include "pch.h"
+#include "StyleTransferEffect.h"
+#include "StyleTransferEffect.g.cpp"
+using namespace std;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::Storage::Streams;
+
+namespace winrt::StyleTransferEffectCpp::implementation
+{
+	StyleTransferEffect::StyleTransferEffect() : outputTransformed(VideoFrame(Windows::Graphics::Imaging::BitmapPixelFormat::Bgra8, 720, 720)),
+		Session(nullptr),
+		Binding(nullptr)
+	{
+	}
+
+	IVectorView<VideoEncodingProperties> StyleTransferEffect::SupportedEncodingProperties() {
+		VideoEncodingProperties encodingProperties = VideoEncodingProperties();
+		encodingProperties.Subtype(L"ARGB32");
+		return single_threaded_vector(std::move(std::vector<VideoEncodingProperties>{encodingProperties})).GetView();
```

Comment:
Just FYI, std::move doesn't make a difference here (it's already moving anyways, since you're giving std::vector as a rvalue)